lighthouse-cli/test/smokehouse/dbw-config.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
       'render-blocking-resources',
       'errors-in-console',
       'efficient-animated-content',
+      'apple-touch-icon', // pull in apple touch icon to test `LinkElements`
```

Comment:
any reason to do this in dbw and not in perf or seo where LinkElements are already being used?

Reply:
just that dbw felt like the dumping ground of error catching appropriate for this sort of thing 😆 

the others are so focused it felt mean to pollute them with SVG nonsense